src/main/java/cz/xtf/radanalytics/oshinko/deployment/template/BaseTemplateDeployment.java

```diff
@@ -0,0 +1,36 @@
+package cz.xtf.radanalytics.oshinko.deployment.template;
+
+import cz.xtf.openshift.OpenShiftUtil;
+import cz.xtf.openshift.OpenShiftUtils;
+import cz.xtf.radanalytics.util.TestHelper;
+import cz.xtf.radanalytics.waiters.OpenshiftAppsWaiters;
+import io.fabric8.openshift.api.model.Template;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+
+@Slf4j
+public class BaseTemplateDeployment {
+	private static final String WORK_DIR = "oshinko_template";
+	private static final OpenShiftUtil openshift = OpenShiftUtils.master();
+
+	protected static void deploy(String templateResourcePath, String templateResourceFileName,Map<String, String> params){
```

Comment:
reformat the code please:  String templateResourceFileName,Map<String, String> 

Would be beneficial to mark this method as public. If somebody would like to deploy custom spark template.